coops-ui/public/config/configuration.json

```diff
@@ -1,7 +1,7 @@
 {
-  "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/",
```

Comment:
Remember to restore this this when you're done testing locally. The expectation is that `local-configuration.json` is used when running the API locally.